# User command to control safety brake system

std_msgs/Header header

bool safety_brake_cmd        # Set command to true to apply the brakes
